Sources/JavaScriptKit/FundamentalObjects/JSClosure.swift

```diff
@@ -1,9 +1,6 @@
 import _CJavaScriptKit
 
-fileprivate var closureRef: JavaScriptHostFuncRef = 0
-fileprivate var sharedClosures: [JavaScriptHostFuncRef: ([JSValue]) -> JSValue] = [:]
-
-/// JSClosureProtocol abstracts closure object in JavaScript, whose lifetime is manualy managed
+/// JSClosureProtocol abstracts closure object in JavaScript, whose lifetime might be manualy managed
```

Comment:
a little more thorough but perhaps not entirely correct:

```suggestion
/// JSClosureProtocol wraps Swift closure objects for use in JavaScript. Conforming types are responsible for managing the lifetime of the closure they wrap, but can delegate that task to the user by requiring an explicit `release()` call.
```